oauth2client/client.py

```diff
@@ -1222,6 +1229,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        # This import needs to be here to avoid a circular dependency.
```

Comment:
`TODO(issue 388): eliminate the circularity that is the reason for this non-top-level import.`